File: primeN_interval.py
import math
start_number = 1
finish_number = 20
#index is the sum of all dividers for the number given
index = 0
for start_number in range(start_number,finish_number+1):
    for counter in range(1, start_number+1):
        if start_number/counter == math.trunc(start_number/counter):
            index = index + 1
    if index > 2 or start_number == 1:
        #don't print, because it isn't a prime number and reset index
        index = 0
    else:
        print(start_number)
        index = 0

File: primeN_interval.js
let start_number = 2;
let finish_number = 20;
let counter = 0;
for(start_number; strat_number < finish_number; start_number++){
  //i is divider for every number. It's there to help the counter
  for(i = 1; i<= start_number; i++){
    if(start_number/i == Math.trunc(start_number/i)){
      counter++;
    }
  }
if(counter > 2 || start_number == 1){
    //print nothing and reset counter
    counter = 0;
}
else{
    console.log(start_number);
    counter = 0;
  }
}
